lib/cleanup.js

```diff
@@ -1,12 +0,0 @@
-function removeLeadingZerosFromStreet(token) {
```

Comment:
Realizing my comment probably wasn't very clear. I'd leave this split out in this file as it was. Not bother making a separate module until future implementation warrants it. I like that it was independent of other things and of `stream` code as well.